odl/util/testutils.py

```diff
@@ -205,7 +205,12 @@ def is_subdict(subdict, dictionary):
     import pytest
 
 except ImportError:
-    from odl.util import OptionalArgDecorator as identity
+
+    def identity(*args, **kwargs):
```

Comment:
```
@identity(apply=lambda x: x+1)
def f(x):
    return 2 * x
```

This would fail, but perhaps that's expected?

Reply:
With the latest version it would "work" (do nothing).

Reply:
I guess the danger is 

```
@identity(lambda x: x+1)
def f(x):
    return 2 * x
```

But perhaps we just leave that be